RadixWallet/Features/AccountDetailsFeature/Coordinator/AccountDetails+View.swift

```diff
@@ -201,6 +206,51 @@ private extension View {
 			AssetTransfer.SheetView(store: $0)
 		}
 	}
+
+	private func fungibleDetails(with destinationStore: PresentationStoreOf<AccountDetails.Destination>) -> some View {
+		sheet(
+			store: destinationStore,
+			state: /AccountDetails.Destination.State.fungibleDetails,
+			action: AccountDetails.Destination.Action.fungibleDetails,
+			content: { FungibleTokenDetails.View(store: $0) }
+		)
```

Comment:
Use the new style, like here:
```swift
		navigationDestination(store: destinationStore.scope(state: \.configurationBackup, action: \.configurationBackup)) {
			ConfigurationBackup.View(store: $0)
		}
```

Reply:
That style is also deprecated actually, we will need to still change it once we move from `ViewStore` to `Bindable`.

Anyway, updated it because it takes less lines at least.

Reply:
Yes sure, but it avoids using the hacky old run-time CasePaths